src/ElementStatusChange.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace putyourlightson\elementstatusevents;
+
+use craft\console\Application;
+use craft\web\Application as CraftWebApp;
+use craft\console\Application as CraftConsoleApp;
+use putyourlightson\elementstatusevents\commands\ScheduledElements;
+use yii\base\BootstrapInterface;
+use yii\base\Component;
+use craft\base\Element;
+use craft\events\ElementEvent;
+use craft\services\Elements;
+use putyourlightson\elementstatusevents\behaviors\ElementStatusBehavior;
+use yii\base\Event;
+
+/**
+ * Class ElementStatusChange
+ *
+ * @package putyourlightson\elementstatusevents
+ */
+class ElementStatusChange extends Component implements BootstrapInterface
+{
+
+    const EVENT_STATUS_CHANGED = 'statusChanged';
+
```

Comment:
> Why renaming the main class?

The class + event constant is the public API for the consumer.
Detecting the "status **change**" is the only purpose of the package. Being more specfic (`ElementStatusChange` vs `ElementStatusEvents`) felt bit better to me. 
However, I'm not super opinionated about that.